etc/upgrades/1711451944-hyperchain-upgrade/stage-proofs-fix2/crypto.json

```diff
@@ -0,0 +1,11 @@
+{
+  "verifier": {
+    "address": "0x137935Cb6dA4b0698dc52e4D274BE64FD8423201",
+    "txHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+  },
+  "keys": {
+    "recursionNodeLevelVkHash": "0xf520cd5b37e74e19fdb369c8d676a04dce8a19457497ac6686d2bb95d94109c8",
+    "recursionLeafLevelVkHash": "0xdb1559fec7b0a02d745c1775061e8565802234d96c6e4278c4129bd11209b33e",
+    "recursionCircuitsSetVksHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
are these still needed since they're baked into the verifier contract?

Reply:
I think the server still gets them from the diamond contract, that what somebody told me.